client/app/docs/views/main.html

```diff
@@ -967,6 +967,13 @@ <h3 translate>Sample modal</h3>
                             scope.closeModal = function() {
                                 scope.isModalOpen = false;
                             };
+                            
+                            $(document).on('hidden.bs.modal', '.modal', function () {
+                                scope.modalActive = false;
+                                if (!scope.$$phase) {

```

Comment:
Don't do that, it's considered an anti-pattern.  Variables names prefixed by `$$` are internal to Angular and their behaviour/existence can change without notice.

Check this link for an idea how to overcome the problem (the last section):
https://github.com/superdesk/superdesk-docs/blob/master/good-practices/angular-performance-tips.md